pkg/cassandra/config.go

```diff
@@ -0,0 +1,43 @@
+package cassandra
+
+import (
+	"encoding/json"
+	"github.com/Jeffail/gabs"
+	"strings"
+)
+
+type NodeConfig map[string]interface{}
+
+func getOperatorSuppliedConfig(dcs []string) NodeConfig {
+	return NodeConfig{
+		"jvm-options": NodeConfig{
```

Comment:
This won't work with 4.0 ? In the config-builder it uses ``jvm-server-options`` for 4.0 and ``jam-options`` for 3.11.x

Reply:
Yeah, I need to add logic for this. The PR is definitely a work in progress :)